backend/fms_core/templates.py

```diff
@@ -108,7 +110,7 @@
       },
       {
           'name': 'Library',
-          'headers': ['Library Batch ID', 'Library Source Container Barcode', 'Library Source Container Coordinates',
+          'headers': ['Library Batch ID', 'Library Source Name', 'Library Source Container Barcode', 'Library Source Container Coordinates',
```

Comment:
"Library Source Name" ... i think we can do better "Source Library Name" is already better. It could also just mention "Source Sample Name" since it is a sample.

Reply:
I kept it like this for consistency.  All of the header are "Library Source ... "